src/Illuminate/Foundation/Vite.php

```diff
@@ -655,6 +655,30 @@ public function asset($asset, $buildDirectory = null)
         return $this->assetPath($buildDirectory.'/'.$chunk['file']);
     }
 
+    /**
+     * Get the content for an asset.
+     *
+     * @param  string  $asset
+     * @param  string|null  $buildDirectory
+     * @return string
+     *
+     * @throws \Exception
+     */
+    public function content($asset, $buildDirectory = null)
+    {
+        $buildDirectory ??= $this->buildDirectory;
+
+        $chunk = $this->chunk($this->manifest($buildDirectory), $asset);
+
+        $path = public_path($buildDirectory.'/'.$chunk['file']);
+
+        if (! is_file($path) || ! file_exists($path)) {
+            throw new Exception("Unable to locate file at path: {$path}.");
```

Comment:
I wonder if including "Vite" might help.

E.g.

```suggestion
            throw new Exception("Unable to locate file from Vite manifest: {$path}.");
```

Hopefully, it's distinct enough from the error when the file doesn't exist in the manifest:

```php
            throw new Exception("Unable to locate file in Vite manifest: {$file}.");
```